source/_integrations/brother.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Brother Printer"
+description: "Instructions on how to integrate a Brother printer into Home Assistant."
+logo: brother.png
+ha_category:
+  - System Monitor
+ha_release: 0.105
+ha_iot_class: Local Polling
+---
+
+The `Brother Printer` integration allows you to read current data from your local Brother printer.
+
+It usually provides information about the device's state, the left amount of ink or toner and remaining life of drum or other parts of the printer.
+The integration monitors every supported part.
+
+## Configuration
+
+To add `Brother Printer` to your installation, go to **Configuration** >> **Integrations** in the UI, click button with `+` sign and from the list of integrations select **Brother Printer**.
+
+{% configuration %}
```

Comment:
Since this integration uses a config flow only approach, we should remove this config block to avoid confusing configuring it via YAML.

Reply:
Done